src/components/Sum.jsx

```diff
@@ -1 +1,31 @@
 // Don't forget to add default values an check types of props
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export function Sum(props) {
```

Comment:
```suggestion
export function Sum({ a, b }) {
```
Можна зразу ж використати деструктуризацію